sysdeps/linux/x86_64/generic.c

```diff
@@ -1,8 +1,14 @@
+#include <stdarg.h>
```

Comment:
That include is useless

Reply:
It'll be useful when making a proper `syscall` function with variadic arguments

Reply:
For the time being it's not used but I totally agree with you